### PR TITLE
ref(*): change RC name to be router inside the deis namespace

### PR DESCRIPTION
**Pull request by helgi**
Required for https://github.com/deis/charts/pull/252

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -416,10 +416,10 @@ If using a Kubernetes distribution or underlying infrastructure that does not su
 
 This manually replicates the configuration that would be achieved automatically with some distributions on some infrastructure providers, as discussed above.
 
-First, determine the "node ports" for the `deis-router` service:
+First, determine the "node ports" for the `deis router` service:
```
Comment (krancour) [inline README.md]: deis should probably appear outside (before) the backticks.

**Comment by krancour** [comment]
Apart from one non-blocking nit, LGTM.

File: README.md (hunk 1 above)
Comment (helgi) [inline README.md]: Fixed

**Comment by krancour** [comment]
I think this cannot pass CI until https://github.com/deis/charts/pull/252 passes-- and unfortunately, vice versa.

**Comment by jchauncey** [comment]
@helgi im guessing we aren't doing this anymore?

**Comment by helgi** [comment]
Punting it to later - going to see if it is worth updating env vars in all repos to function with the chart updates

**Comment by krancour** [comment]
@helgi, should this be closed, especially now that any remaining RC's are vestigial?

**Comment by mboersma** [comment]
@helgi is this still in progress or should we close it?

**Comment by helgi** [comment]
Closing - doesn't seem feasible to pull off anytime soon

**Pull request closed without merging**